docs/topics/vs-code-extension-run-configuration.adoc

```diff
@@ -14,7 +14,11 @@ You can configure multiple run configurations to run against each project you im
 . Click the *+* (plus sign) next to *Migration Toolkit for Applications* to add a run configuration.
 . Complete the following configuration fields:
 
-** *Input*: Click *Add* and do one of the following:
+** *Name*: Enter a meaningful name for the analysis configuration or accept the default.
+** *`mta-cli`*: Enter the path to the `mta-cli` executable. For example: `$HOME/mta-cli-5.3.0.Final/bin/mta-cli`.
++
+You can download the `mta-cli` executable from link:https://developers.redhat.com/products/mta/download[MTA download].
```

Comment:
@RichardHoch 
It seems to me that having MTA CLI executable should be a prerequisite for this procedure. Something like "You need to have the MTA CLI executable available. If you do not have it,  download it from ..."

Reply:
@sbeskin-redhat Good point. I made it a prerequisite.